master/getting-started/kubernetes/installation/other.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Installing Calico on Kubernetes with other networking (advanced)
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/thirdpty'
+---
+
+If you do not want to use {{site.prodname}} or flannel for networking, you
```

Comment:
rather than "can also route traffic based on pod allocations" this should probably say "you can also enforce Calico policy on top of other networking solutions, for example"